readme.html

```diff
@@ -1,10 +1,30 @@
 <html>
   <head></head>
   <body>
-    <h1>Code42 Phantom App</h1>
-    <p>This app implements various investigative actions on the Code42 Incydr platform.</p>
+    <h2>py42</h2>
```

Comment:
It is common in phantom to list dependencies first in this file.

It is a great bonus because we get to mention our SDK!